tests/integration/test_s3.py

```diff
@@ -110,19 +125,31 @@ def typical_workflow(
     print("Done.")
 
 
-def test_typical_workflow(s3_fixture):  # noqa: F811
+@pytest.mark.parametrize("use_multipart_upload", [True, False])
+def test_typical_workflow(use_multipart_upload: bool, s3_fixture):  # noqa: F811
     """
     Tests all methods of the ObjectStorageS3 DAO implementation in one long workflow.
     """
-    object_fixture = s3_fixture.non_existing_objects[0]
-    typical_workflow(
-        storage_client=s3_fixture.storage,
-        bucket1_id=s3_fixture.non_existing_buckets[0],
-        bucket2_id=s3_fixture.non_existing_buckets[1],
-        object_id=object_fixture.object_id,
-        test_file_md5=object_fixture.md5,
-        test_file_path=object_fixture.file_path,
-    )
+    with (
+        big_temp_file(size=20 * pow(10, 6)) if use_multipart_upload else nullcontext()
```

Comment:
Since S3 always uses MiB instead of MB, i suggest we do the same and use MiB throughout the entire project to avoid confusion or incompatibilities.

```suggestion
        big_temp_file(size=20 * pow(2, 20)) if use_multipart_upload else nullcontext()
```